### PR TITLE
separate igo and unclaimed on fight rewards

### PR DESCRIPTION
**Pull request by byyyyyyyyy**
### All Submissions

* [x] Can you post a screenshot of your changes (if applicable)?
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### Testing

Has the code been tested, or does it need double checking?

### CODE REVIEW

File: frontend/src/components/CombatResults.vue [hunk 1]
```diff
@@ -39,7 +39,8 @@
         <b-row>
           <b-col class="earned">
             <h4>
-              {{formattedSkill}}  {{formattedUsd}}
+              {{formattedSkillNoIGO}} {{formattedInUsd(calculateSkillPriceInUsd(calculateSkillRewardNoIGO()).toFixed(4))}} UNCLAIMED SKILL AND
```
Comment (Sellie96) [inline frontend/src/components/CombatResults.vue:42]: Translations

File: frontend/src/store/store.ts [hunk 2]
```diff
@@ -2019,6 +2028,8 @@ export default new Vuex.Store<IState>({
         )
         .send({ from: state.defaultAccount, gas: '300000', gasPrice: getGasPrice() });
 
+      console.log(res);
```
Comment (Sellie96) [inline frontend/src/store/store.ts:2031]: Dev log

File: frontend/src/components/CombatResults.vue [hunk 3]
```diff
@@ -39,7 +39,8 @@
...
+              {{formattedSkillIGOReward}} {{formattedInUsd(calculateSkillPriceInUsd(calculateSkillIGOReward()).toFixed(4))}} IGO
```
Comment (Sellie96) [inline frontend/src/components/CombatResults.vue:43]: Is the case handled if IGO isn't returned after the event? I'd rather not have to delete this back out afterwards?

**Comment by Sellie96** [comment]
Could also do with better formatting of the messages. Little bit hard to read at a glance there

**Review by kevinunger — APPROVED** [review]
lgtm!

**Pull request merged**